napari_arboretum/plugin.py

```diff
@@ -111,3 +118,18 @@ def draw_current_time_line(self, event: Event | None = None) -> None:
         z_value = self.viewer.dims.current_step[0]
         self.plotter.draw_current_time_line(z_value)
         self.property_plotter.draw_current_time_line(z_value)
+
+    def export_tree(self) -> None:
+        """Export the tree as an SVG."""
+        root_id = get_root_id(self.tracks, self.track_id)
+        options = QFileDialog.Options()
+        options |= QFileDialog.DontUseNativeDialog
```

Comment:
Why have these on separate lines?

Reply:
I've actually removed the second line, so that we do use the native dialog now.